aegisub-flatpak-appdata.patch

```diff
@@ -0,0 +1,15 @@
+diff -Naur aegisub-flatpak.orig/org.aegisub.Aegisub.appdata.xml aegisub-flatpak/org.aegisub.Aegisub.appdata.xml
+--- aegisub-flatpak.orig/org.aegisub.Aegisub.appdata.xml	2018-12-06 03:18:01.115323000 +0100
++++ aegisub-flatpak/org.aegisub.Aegisub.appdata.xml	2018-12-06 19:30:41.135125735 +0100
+@@ -65,9 +65,8 @@
+   <url type="help">http://docs.aegisub.org</url>
+   <url type="homepage">http://www.aegisub.org</url>
+   <url type="translate">https://sites.google.com/site/rockytdrontransifex/aegisub</url>
+-  <content_rating type="oars-1.0">
+-    <content_attribute id="social-info">mild</content_attribute>
```

Comment:
Why this was removed?

Reply:
> Why this was removed?

Not removed, but replaced. And I have already explained this in the comment.

https://github.com/scx/flathub/blob/d2d77a8d24dfc89ceea426b11ffa1d55baf5111b/Aegisub.yaml#L48-L51
```
  # Update AppData file
  # Change the OARS rating due to disabling the update checker
  - type: patch
    path: aegisub-flatpak-appdata.patch
```

See also:
https://github.com/scx/flathub/blob/d2d77a8d24dfc89ceea426b11ffa1d55baf5111b/Aegisub.yaml#L2-L3
```
config-opts:
  - --disable-update-checker
```